students/10th/taesung/westagram/user/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render
+import json
+from django.views import View
+from django.http import JsonResponse
+from .models import User
+
+class signUp(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            User(
+                    name = data['name'],
+                    email = data['email'],
+                    password = data['password']
+                    ).save()
+        except Exception as e:
+            return JsonResponse({"message": f"{e}"}, status=400)
+
+        return JsonResponse({'message':'SUCCESS'}, status=200)
+    def get(self, request):
+        user_data = user.objects.values()
+        return JsonResponse({'user':list(user_data)}, status=200)
+
+class signIn(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            i = data['name']
```

Comment:
```suggestion
            name = data['name']
```
변수명을 제대로 작성하는 습관을 가지도록 해주세요.